ote/src/cljs/ote/ui/form_fields.cljs

```diff
@@ -777,44 +780,53 @@
 (defmethod field :default [opts data]
   [:div.error "Missing field type: " (:type opts)])
 
-
-(defmethod field :div-table [{:keys [table-fields div-class update! error-data delete? delete-label
-                                     add-label add-label-disabled? id] :as options} data]
+(defmethod field :div-table [{:keys [table-fields div-class update! error-data delete? inner-delete? delete-label
+                                     inner-delete-label add-label add-inner-label add-label-disabled? id add-divider?] :as options} data]
   (let [data (if (empty? data)
                ;; div-table always contains at least one row
                [{}]
                data)]
-    [:div
+    [:div {:key (str "main-div-table-" id)
+           :style {:clear "both"}}
```

Comment:
Is clear absolutely needed and if yes, could same be achieved using more modern and maintainable css approaches?
Asking because in the olden days float was (mis)used for layouting purposes, often resulting in other problems. `clear` property was often used to work around some float issues. And float shouldn't be used nowadays :)

Reply:
It is not. class row handles the new row part. I'll remove this.